toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,578 @@
+/*global data, labelSummary, imageSummary, jQuery, $, modelHistories*/
+
+// eslint-disable-next-line no-unused-vars
+function openNav() {
+    document.getElementById("mySidenav").style.width = "250px";
+    document.getElementById("main").style.marginLeft = "250px";
+}
+
+// eslint-disable-next-line no-unused-vars
+function closeNav() {
+    document.getElementById("mySidenav").style.width = "0";
+    document.getElementById("main").style.marginLeft = "0";
+}
+
+// eslint-disable-next-line no-unused-vars
+function myreload() {
+    location.reload();
+}
+
+
+
+function insertIntoTopKTable() {
+    var table = document.getElementById("top-5-match-table");
+    var row = table.insertRow(-1);
+    var row2 = table.insertRow(-1);
+
+    data.topKStats.forEach(function (element) {
+        var cell = row.insertCell(-1);
+        cell.innerHTML = "<b>" + element.matches + "</b>";
```

Comment:
Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)